misc/python/materialize/test_analytics/config/test_analytics_db_config.py

```diff
@@ -0,0 +1,18 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import os
+
+MATERIALIZE_PROD_SANDBOX_HOSTNAME = os.environ["MATERIALIZE_PROD_SANDBOX_HOSTNAME"]
```

Comment:
I think I will need to rework this part here.

@def- What is the recommended way to get the host name? Do I need to follow an approach as in `test/canary-load/mzcompose.py` or does a more "light-way" approach exist?

Reply:
Okay, will do 👍

Reply:
=> https://github.com/MaterializeInc/materialize/pull/27442

Reply:
I did not touch canary-environment since it is handled there in a different way. Though, I can if we want to align the approach.